app/views/profiles/_edit.haml

```diff
@@ -9,10 +9,22 @@
   = render 'profiles/edit_public', :profile => profile, :aspect => aspect, :person => person
 
   %hr
+  :javascript
```

Comment:
I'm guessing that `.settings_visibility` does not exist in `profile/edit` and it only exists under `user/edit`